src/test/kotlin/com/terraformation/backend/db/DatabaseTest.kt

```diff
@@ -1721,17 +1727,42 @@ abstract class DatabaseTest {
             modifiedBy = createdBy.toIdWrapper { UserId(it) },
             modifiedTime = createdTime,
             name = name,
-        )
+            cohortId = cohortId?.toIdWrapper { CohortId(it) })
 
     participantsDao.insert(row)
 
     return row.id!!.also { inserted.participantIds.add(it) }
   }
 
+  private var nextCohortNumber = 1
+
+  fun insertCohort(
+      id: Any? = null,
+      name: String = "Cohort ${nextCohortNumber++}",
+      phase: CohortPhase = CohortPhase.Phase0DueDiligence,
+      createdBy: Any = currentUser().userId,
+      createdTime: Instant = Instant.EPOCH,
+  ): CohortId {
+    val row =
+        CohortsRow(
+            createdBy = createdBy.toIdWrapper { UserId(it) },
+            createdTime = createdTime,
+            id = id?.toIdWrapper { CohortId(it) },
+            modifiedBy = createdBy.toIdWrapper { UserId(it) },
+            modifiedTime = createdTime,
+            name = name,
+            phaseId = phase)
```

Comment:
Nit: Maybe throw a trailing comma on here just so future diffs that add more columns to this table don't have to change this line.